src/fftpack.f90

```diff
@@ -268,6 +270,46 @@ pure module function idct_rk(x, n, type) result(result)
         end function idct_rk
     end interface idct
 
+    !> Version: experimental
+    !>
+    !> Initialize DCT type-1
+    !> ([Specification](../page/specs/fftpack.html#dct_t1i))
```

Comment:
```suggestion
    !> ([Specification](../page/specs/fftpack.html#initialize-dct-1-dcosti-or-dct_t1i))
```

This should be indexed to the ford-style title, but I don't think it's a big problem, we can submit a clean PR at the end to link the document title correctly.

Reply:
Oh I see, I'll make the changes and we can merge this PR then, so that there's no extra work pending.

Reply:
Yes, it seems to be so.